### PR TITLE
[risk=low][RW-10024] bump e2e test user to 8

### PR DESCRIPTION
**Pull request by jmthibault79**
**Description**

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
- [ ] If this PR is intended to complete a JIRA story, I have checked that all AC are met for that story
- [ ] I have manually tested this change and my testing process is described above
- [ ] This PR includes appropriate automated tests, and I have documented any behavior that cannot be tested with code
- [ ] If this fixes a bug, ensure the steps to reproduce are in the Jira ticket or provided above.
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented the impacts in the description
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this includes an API change, I have run the relevant E2E tests locally because API changes are not covered by our PR checks

### CODE REVIEW

File: api/config/config_test.json [hunk 1]
```diff
@@ -172,13 +172,20 @@
   "e2eTestUsers": {
     "workspaceDeletionBatchSize": 15,
     "testUserEmails": [
+      "puppeteer-tester-1@fake-research-aou.org",
```
Comment (jmthibault79) [inline api/config/config_test.json:175]: This list is for the deletion scripts.  I thought it would be good to clean up older users too.

Comment (evrii) [reply api/config/config_test.json:175]: How many users did you add? It looks like the old version of the file did not include puppeteer-tester-1, puppeteer-tester-2, or puppeteer-tester-4 through puppeteer-tester-6.

**Review by evrii — APPROVED** [review]
Looks good to me.

File: api/config/config_test.json (hunk 1 above)
Comment (jmthibault79) [reply api/config/config_test.json:175]: for each user whatever-N, I added 1 to N here

**Pull request merged**